_source/_posts/2021-11-01-saml-assertion-flow-1.adoc

```diff
@@ -0,0 +1,264 @@
+---
+layout: blog_post
+title: "OAuth SAML 2.0 Assertion Grant Type"
+author: erik-manor
```

Comment:
Needs an entry for Erik in the [authors.yml](https://github.com/oktadev/okta-blog/blob/main/_source/_data/authors.yml) file